onnxruntime/test/shared_lib/test_inference.cc

```diff
@@ -280,6 +288,8 @@ TEST(CApiTest, custom_op_handler) {
   Ort::CustomOpDomain custom_op_domain("");
   custom_op_domain.Add(&custom_op);
 
+  void* library_handle = nullptr;
+
```

Comment:
So, the same thing, we get this handle and what's the purpose?

Reply:
Same as above.

Reply:
Simplified everything - should be much cleaner now